cmd/qosserver/deploy.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: thundernetes-qosserver
+  namespace: thundernetes-system
+  labels:
+    app: thundernetes-qosserver
+spec:
+  selector:
+    matchLabels:
+      app: thundernetes-qosserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: thundernetes-qosserver
+    spec:
+      containers:
+        - image: thundernetes-qosserver:${IMAGE_TAG}
+          name: qosserver
+          imagePullPolicy: IfNotPresent
+          resources:
+            requests:
+              cpu: 100m
+              memory: 500Mi
+            limits:
+              cpu: 1000m
+              memory: 1000Mi
+          ports:
+            - containerPort: ${UDP_SERVER_PORT}
```

Comment:
we can use the default ports (3075/8080) here and in the monitor.yaml file, user can easily change them before deploying